content/outcomes/application-development/prioritizing-outcomes.md

```diff
@@ -0,0 +1,170 @@
+
+**Relative complexity** means thinking about similar stories that the team has delivered in the past and making a relative estimate. This way, the team can build up a consistent, shared idea of what constitutes a small, medium or large-sized story.
+
+When team members give different estimates for a story, this can give rise to valuable conversations! Estimation prompts team members to talk about their assumptions and arrive at a shared understanding of what is involved in the work. Learn more in our [short guide to estimation](https://tanzu.vmware.com/developer/practices/ipm/#estimation).
+
+## Describe How Measuring Velocity Enables Teams to Predict Delivery
+
+> _"When is it going to be done?"_
+>
+> -- Your boss
+
+Every team responsible for a deliverable has been asked this question. 
+
+Software teams commonly need to gauge how long it takes to deliver a feature, or say whether a deadline will be met. Yet haven't we just described how we don't recommend estimating based on time of delivery?
+
+Estimation gives the team _data which helps them learn how to deliver more predictably over time_ and make predictions about future delivery. 
+
+To do this, we use the concept of **velocity**. You can calculate velocity by taking an average of the story points delivered in previous iterations. Once you know how many story points are left in your backlog, you can use your team’s velocity to forecast approximately how many iterations it is going to take to deliver the rest of the work.
+ 
+Of course circumstances change and unexpected things happen, so predictions are never going be 100 percent certain. To accommodate for risk, use a range of estimates based on optimistic and pessimistic scenarios. 
+
+If the number of story points a team delivers varies widely across multiple iterations, then the team’s velocity has **high volatility**. This makes it harder to predict delivery. It’s worth taking time with your team to understand why volatility is high and try to address it. Even uncertain estimates help to reduce and communicate risk, and make software delivery more predictable. Again, predictability and consistency are more important than raw numbers, and enable more accurate future planning.
+
+## In this Article, You Learned to
+
+* Define what a user story is and why they are useful.
+* Describe the advantages of estimating work in terms of risk and complexity instead of time.
+* Describe how measuring velocity enables teams to predict delivery
+
+
+## Keep Learning
+
+- Want to learn more about balanced teams? Check out the talk, [“Making magic with Balanced Teams”](https://tanzu.vmware.com/content/videos/becki-hyde-making-magic-with-balanced-teams) at SpringOne by  Tanzu Labs Product Manager Becki Hyde.
```

Comment:
Find out how to run an [iteration planning meeting](https://tanzu.vmware.com/developer/practices/ipm/), where your team will estimate and discuss user stories.